StantDrawing/BrushTool/BrushToolView.swift

```diff
@@ -187,27 +221,115 @@ public class BrushToolView: UIView {
     }
     
     private func configureGroupPencilSizeLayout() {
-        let origin = CGPoint(x: groupToolsView.frame.width + groupToolsView.frame.origin.x + 10, y: 0)
-        configureGroupLayout(buttons: groupPencilSizeButtons, groupView: groupPencilSizeView, groupViewOirigin: origin, addAsSubview: false)
-        groupPencilSizeView.isHidden = true
+        // Pencil Icon
+        let pencilIconView   = UIView()
         
-        setAllGroupdPencilSizeButtonsAsNotClicked()
+        let originIconView = CGPoint(x: 0, y: 0)
+        let sizeIconView   = CGSize(width: groupViewWidth, height: 60)
+        let frameIconView  = CGRect.init(origin: originIconView, size: sizeIconView)
+        
+        pencilIconView.frame              = frameIconView
+        pencilIconView.backgroundColor    = UIColor(hex: "F0F0F0")
+        pencilIconView.layer.cornerRadius = pencilIconView.frame.width / 2
+        
+        let originIconImg = CGPoint(x: (groupViewWidth - 25) / 2, y: 18)
+        let sizeIconImg   = CGSize(width: 25, height: 25)
+        let frameIconImg  = CGRect.init(origin: originIconImg, size: sizeIconImg)
+        
+        let pencilIconImageView = UIImageView(image: UIImage(named: "pencil", in: bundle, compatibleWith: nil))
```

Comment:
This would follow the same idea that I stated before about the color extension (in this case, would be an image extension).

Reply:
I don't believe in this case is necessary. We have only 1 line using it.